### PR TITLE
Changed base postgres version to 16 and added automated docker build

### PR DESCRIPTION
**Pull request by mawoka-myblock**
I think the title says everything relevant.

### CODE REVIEW

**Comment by craigpastro** [comment]
Sorry! Somehow I missed this. I’ll try to take a look this week. Thank you!

**Comment by mawoka-myblock** [comment]
Here is the link to the rocker image built by the script above:

https://github.com/mawoka-myblock/pg_uuidv7/pkgs/container/pg_uuidv7

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ "main" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
```
Comment (craigpastro) [inline .github/workflows/docker-publish.yml:13]: Just a question, but could we get rid of the “if pull request” checks if we dropped this?

**Review by craigpastro — COMMENTED** [review]
This looks good to me! I left one question. I also have one nitty ask: could you rename the docker-publish file to use .yaml as the extension. Sorry, it’s my pet peeve. Thank you for the contribution!

File: .github/workflows/docker-publish.yml (hunk 1 above)
Comment (mawoka-myblock) [reply .github/workflows/docker-publish.yml:13]: Probably. Will check that in 1 hour

**Comment by craigpastro** [comment]
Thanks again for the contribution ❤️ By the way, do you think we should add something to the readme as it is now easier to test out the extension?

**Comment by mawoka-myblock** [comment]
Yea, adding the note regarding docker is probably a good idea. Also don't forget to set the package to public.

**Pull request merged**